tests/examples/awful/widget/layoutbox/default.lua

```diff
@@ -0,0 +1,48 @@
+--DOC_NO_USAGE --DOC_GEN_IMAGE
+local awful     = require("awful") --DOC_HIDE
+local wibox     = require("wibox") --DOC_HIDE
+
+screen[1]._resize {width = 480, height = 100} --DOC_HIDE
+
+local wb = awful.wibar { position = "top", } --DOC_HIDE
+
+--DOC_HIDE Create the same number of tags as the default config
+awful.tag({ "1", "2", "3", "4", "5", "6", "7", "8", "9" }, screen[1], awful.layout.layouts[1]) --DOC_HIDE
+
+--DOC_HIDE Only bother with widgets that are visible by default
+local mykeyboardlayout = awful.widget.keyboardlayout() --DOC_HIDE
+local mytextclock = wibox.widget.textclock() --DOC_HIDE
+local mytaglist = awful.widget.taglist(screen[1], awful.widget.taglist.filter.all, {}) --DOC_HIDE
+local mytasklist = awful.widget.tasklist(screen[1], awful.widget.tasklist.filter.currenttags, {}) --DOC_HIDE
+
+--DOC_NEWLINE
+
+    local mylayoutbox = awful.widget.layoutbox {
+        s = screen[1],
```

Comment:
The required argument for the screen is named `screen` not `s`. It works here because `awful.widget.layoutbox` fallback to screen 1 but such an error shouldn't stay in example code.
```suggestion
        screen = screen[1],
```

BTW, to mimic at best the code from the `awesomerc.lua` I suggest to change the screen reference to `s` earlier on this script. So this line would be exactly the same than in the default rc:

```suggestion
        screen = s,
```

Reply:
done